keyboards/handwired/oem_ansi_fullsize/info.json

```diff
@@ -0,0 +1,19 @@
+{
+    "keyboard_name": "oem_ansi_fullsize",
+    "url": "",
+    "maintainer": "Cian Johnston",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This doesnt match the layout macro so will be spitting out api/`qmk info` issues

Reply:
Is there a nice way to generate this from the existing source code, or would I be better off just adapting the `info.json` of `handwired/oem_iso_fullsize`?

Reply:
> EDIT: yep, qmk c2json

Nope, thats for a different purpose. https://qmk.fm/converter/ or use `layouts/default/fullsize_iso/info.json` as a base.

Reply:
Done